.github/workflows/docs.yml

```
@@ -7,6 +7,12 @@ on:
       - opened
       - reopened
       - synchronize
+    paths_ignore:
```

```suggestion
    paths-ignore:
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths